book/plugins/android/single-bundle.md

```diff
@@ -0,0 +1,34 @@
+## Single Bundle React Naitve
+
+![single-bundle.png](./single-bundle.png)
+
+### OVERVIEW
+This plugin gives ability to change version of react native and react. It also aggregate all react native plugins into one single bundle and upload it to S3. Plugins becomes modules under the names specified as plugin identifiers from plugin configuration."
```

Comment:
I think that the main motivation to use this plug-in is that it uses a single bundle that is more efficient and easy to handle in a native app so I would start with something like that. I would then mention that by using it it provides the flexibility to control react and rn versions